pkg/replicator/replicator.go

```diff
@@ -98,6 +98,10 @@ func (d *dbImpl) ChangeReplicationSource(host, replicaPass string, port int32) e
 }
 
 func (d *dbImpl) StartReplication(host, replicaPass string, port int32) error {
+	if err := d.StopReplication(); err != nil {
```

Comment:
Maybe I need to move it to bootstrap/main.go ... What do you think?

Reply:
it may be a good idea but please provide a description, why does this fix the issue?

Reply:
@egegunes I have updated description